### ISSUE TITLE
Proper incremental builds

### ISSUE THREAD

**Issue by zenhack** [capnproto/ekam#24]
Per the readme:

> Note that continuous building is the only way to do incremental builds with Ekam -- any time you run a new Ekam process, it always starts from scratch. I generally just leave Ekam running in a console window 24/7.

Leaving ekam running seems like a reasonable strategy for a workstation that you can leave on anyway, but I generally work from my laptop, and would prefer not to have to start a cold build each time I boot up the machine.

So it would be nice to have proper incremental builds that don't require a continuously running ekam.

**Comment by kentonv** [comment]
Yeah, big project though. Basically we'd need to serialize the internal state to a file that we could reload later.
